src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/DelegatingVideoLinkBookingEventListener.kt

```diff
@@ -38,6 +38,9 @@ class DelegatingVideoLinkBookingEventListener(
     applicationInsightsEventListener.bookingDeleted(copy)
   }
 
+  override fun bookingUpdatedInNomis(booking: VideoLinkBooking) {
+  }
```

Comment:
this will need to call 
 `applicationInsightsEventListener. bookingUpdatedInNomis(copy)`